### DISCUSSION TITLE
I implemented tRpc in Remix, remix doesnt reload loader after mutation

### DISCUSSION THREAD

**Discussion by tomer-yechiel** [remix-run/remix#7999] [Q&A] [answered]
does anyone sort it out?
the reason i want tRpc, is the async await api it has and fetcher lack

**Comment by sergiodxa** [comment, marked as answer]
You will need to use the useRevalidator hook to trigger a revalidation manually

**Reply by tomer-yechiel** [reply]
thanks! works!
